### ISSUE TITLE
ipython 5 tab completion not compatible with jpype

### ISSUE THREAD

**Issue by rdemaria** [rdemaria/pytimber#36]
New version of ipython probably use threads to implement tab completion that crashes the jvm.

```
import jpype
jpype.startJVM(jpype.get_default_jvm_path())
jpype.java.lang.System.out.<TAB>
```

should crash the JVM. It looks like jupyter-console works as well...

see https://github.com/ipython/ipython/issues/10315

**Comment by levens** [comment]
This is fixed in originell/jpype#209 but is not yet in a PyPI release.

**Comment by levens** [comment]
Hi Riccardo,

Your fix in 9bdcfca4af30b6f9e756f2f16c5f319867f12b62 only seems effective at the first level of the hierarchy. For example:

`ldb.tree.<tab>`

Is OK, whereas:

`ldb.tree.LHC.<tab>`

Crashes. However, the new version of JPype seems to be OK, so hopefully they will make a new release soon.

**Comment by pelson** [comment]
Think this can be closed. (don't have permission to do so)
